tests/test_cmd_meta_validator.py

```diff
@@ -10,10 +9,22 @@ def setUp(self) -> None:
         super().setUp()
         self.validator = CommandMetaValidator('2.50.0', 'test_meta')
 
-    def test_validate_command(self):
-        self.validator.validate_command('az webapp create -g g -n n -p p')
-        self.validator.validate_command('az vm create -n n --resource-group g')
-        with self.assertRaises(ParserFailureException):
-            self.validator.validate_command('az vm create -n n')
-        with self.assertRaises(ParserFailureException):
-            self.validator.validate_command('az vm create -n n -g g --unknown')
+    # Test cases
+    def test_validate_correct_command_with_no_ids(self):
+        self.validator.validate_command('az storage account show -g qinkai-test -n storageqinkai')
+        
+    def test_validate_correct_command_with_ids(self):
+        self.validator.validate_command('az storage account show --ids /subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/qinkai-test/providers/Microsoft.Storage/storageAccounts/storageqinkai')
+
+    def test_validate_correct_command_with_ids_and_separated_parameter_with_id_part(self):
+        self.validator.validate_command('az storage account show --ids /subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/qinkai-test/providers/Microsoft.Storage/storageAccounts/storageqinkai -n storageqin')
```

Comment:
Please use placeholders such as xxxx to mask the sensitive information in the test cases.

Reply:
Will do!